draft-ion.adoc

```diff
@@ -208,79 +200,176 @@ represented as follows:
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
 "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and
-"OPTIONAL" in this document are to be interpreted as described in Key
-words for use in RFCs to Indicate Requirement Levels {uri-rfc2119}[RFC2119].  If
-these words are used without being spelled in uppercase then they are
-to be interpreted with their normal natural language meanings.
+"OPTIONAL" in this document are to be interpreted as described {uri-rfc2119}[RFC2119] when they appear in UPPERCASE.  If these words are used without being spelled in uppercase then they are
+to be interpreted with their normal natural language meanings, without any normative meaning.
+
 
+[[content-structure]]
+== Content Structure
 
+Ion content MUST have exactly one root JSON object, called the _Root Object_.  The Root Object MAY contain
+name/value pairs called 'members' and any level of nested JSON objects.
+
+[[metadata]]
 == Metadata
 
-An Ion resource MAY contain a direct child _Meta Object_. If present,
-the _Meta Object_ represents data about its immediate parent Ion resource.
-The _Meta Object_ is OPTIONAL.
+An JSON object MAY contain a _Meta Object_ member named `meta`. If present, the _Meta Object_ represents data about its immediate parent JSON object. The _Meta Object_ is OPTIONAL.
+
+An Ion parser MUST identify a member as a _Meta Object_ if and only if:
 
-If a _Meta Object_ is present, the resource field name MUST equal the
-case-sensitive octet sequence `meta` and the field value MUST be a non-null, non-empty
-JSON object. A _Meta Object_ MUST contain one or more name/value
-pairs. Ion parsers MUST reject Ion resources where a direct child `meta` field
-exists and the field value is `null` or an empty JSON object.
+1. The member name is equal to the case-sensitive octet sequence `meta` and
+2. The member value is a JSON Object that has one or more members.
```

Comment:
Shouldn't this be:

The member value is a `Meta Object` that has one or more members.